 Gallery crash fixes
fix app close when we select album

diff --git a/src/com/android/gallery3d/app/AlbumPage.java b/src/com/android/gallery3d/app/AlbumPage.java
index 011a979..017102a 100644
--- a/src/com/android/gallery3d/app/AlbumPage.java
+++ b/src/com/android/gallery3d/app/AlbumPage.java
@@ -628,13 +628,15 @@ protected boolean onCreateActionBar(Menu menu) {
             inflator.inflate(R.menu.pickup, menu);
             int typeBits = mData.getInt(GalleryActivity.KEY_TYPE_BITS,
                     DataManager.INCLUDE_IMAGE);
-            mActionBar.setTitle(GalleryUtils.getSelectionModePrompt(typeBits));
+            if (mActionBar != null) {
+                mActionBar.setTitle(GalleryUtils.getSelectionModePrompt(typeBits));
+            }
         } else {
             inflator.inflate(R.menu.album, menu);
-            mActionBar.setTitle(mMediaSet.getName());
-
-            FilterUtils.setupMenuItems(mActionBar, mMediaSetPath, true);
-
+            if (mActionBar != null) {
+                mActionBar.setTitle(mMediaSet.getName());
+                FilterUtils.setupMenuItems(mActionBar, mMediaSetPath, true);
+            }
             menu.findItem(R.id.action_camera).setVisible(
                    GalleryUtils.isAnyCameraAvailable(mActivity));
             menu.findItem(R.id.action_slideshow).setVisible(!mIsVideoScreen);
